test/YankPasteTest.VI.cs

```diff
@@ -324,16 +324,17 @@ public void ViPasteAfterYankBeginningOfLine()
         {
             TestSetup(KeyMode.Vi);
 
-            Test("012", Keys(
```

Comment:
Why would you remove the single line test?

Reply:
No particular reason, indeed.
I re-introduced the one-line test, taking into account the new cursor position change.